src/enter-jam/index.md

```diff
@@ -21,6 +21,8 @@ Making a game page before your game is finished allows people to follow its deve
 
 To enter your game in the jam, **put the jam hashtag in the game's description**. You can edit the description from the game's management page on your [dashboard](http://gamejolt.com/dashboard/). Make sure to add the hashtag after the jam begins, but before the jam ends.
 
+Note that adding a jam hashtag creates an entry page on the jam site that is different from the original game page.
```

Comment:
"Note that adding a jam hashtag creates a **separate** entry page on the jam site that is different from the original game page." 

also should we add something like "Voting will be calculated based on the ratings from the jam entry page, not the original game page"